zeek/README.md

```diff
@@ -0,0 +1,195 @@
+## Overview
+
+[Zeek][6] is the world's leading platform for network security monitoring. It interprets what it sees and creates compact, high-fidelity transaction logs, and file content. It can create fully customized output, suitable for manual review on disk or in a more analyst-friendly tool like a security and information event management (SIEM) system.
+
+This integration provides visualization and log enrichments for Connection Logs, DNS & DHCP Logs, Network Protocols, Files, Detections, and Miscellaneous event types.
+
+## Setup
+
+### Installation
+
+To install the Zeek integration, use the `datadog-agent integration install` command and follow the steps below. For more information, see the [Integration Management][7] documentation.
+
+Linux command
+  ```shell
+  sudo -u dd-agent -- datadog-agent integration install datadog-zeek==1.0.0
+  ```
+
+#### Opensource Zeek
+1. [Install the Agent][4] on your Zeek machine.
+2. Setup json-streaming-logs package
+   - Install [Corelight Zeek plugin][5] for JSON logging
+     ```
+     /opt/zeek/bin/zkg install corelight/json-streaming-logs
+     ```
+   - Load ZKG packages
+     ```
+     echo -e "\n# Load ZKG packages\n@load packages" >> /opt/zeek/share/zeek/site/local.zeek
+     ```
+   - Restart the zeek
+     ```
+     /opt/zeek/bin/zeekctl install
+     ```
+     ```
+     /opt/zeek/bin/zeekctl restart
+     ```
+
+#### Corelight Zeek
+* Have the [Datadog Agent][4] installed and running.
+
+### Configuration
+
+#### Log collection
+
+#### Opensource Zeek
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in `datadog.yaml`.
+    ```yaml
+    logs_enabled: true
+    ```
+
+2. Add this configuration block to your `zeek.d/conf.yaml` file to start collecting your Zeek logs.  
```

Comment:
It would be great to add a link to the example configuration file we have in integrations-core. This is what we do for most of our integrations